Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/shared/set-page-params.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useEffect } from 'react';
+import { useParams } from 'react-router';
+import isNil from 'lodash/isNil';
+
+import { IHaveChildrenProps } from '../../components/common/Props';
+import { useRouteUrlParams } from '../../hooks/common/use-route-url-params';
+
+const Page = ({ children }: IHaveChildrenProps) => {
+    const params = useParams();
+    const { actions: { setParams } } = useRouteUrlParams();
+
+    useEffect(
+        () => {
+            if (isNil(params)) {
+                return;
+            }
+
+            setParams(params);
+        },
+        [ params, setParams ],
+    );
+
+    return (
+        <>
+            {children}
+            {' '}
```

Comment:
I guess this is due to _useless `<>`_. Just remove the `<>`. It should be fine.

If a rule is present, it should be OK to silence it without making workarounds.